examples/todo-mvc-svelte/src/App.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+  import Todos from './Todos.svelte';
```

Comment:
Notice for people who will create Svelte example in the future :smile: 

Funny thing happened: the https://github.com/vitejs/vite/tree/main/packages/create-vite/template-svelte-ts/src provides the folder `lib`, so I put the components and the machines into that folder. Svelte and SvelteKit like generating default `lib` folder.

I did not notice when pushed the commit that the `lib` folder is ignored, until running the project in CodeSandbox with huge errors.

`lib` is ignored here: :smile:  https://github.com/statelyai/xstate/blob/f99163680a385d629ac4314f5b0189709f4b57fa/.gitignore#L30-L35